www/md_base/guanlecoja/config.coffee

```diff
@@ -56,6 +56,9 @@ config =
             'underscore.string':
                 version: "~2.3.3"
                 files: 'lib/underscore.string.js'
+            'sortable.js':
```

Comment:
This lib does not depend on jQuery, and it supports drag and sort on `div`s
with `float: left` even if they are all in different sizes. What's more, it
uses HTML5 drag API and works better than that doesn't.

It officially ships with an angular binding and best conforms to what I
need. That's why I finally chose it.

On Sun, May 17, 2015 at 4:31 PM Pierre Tardy <notifications@github.com>
wrote:

> In www/md_base/guanlecoja/config.coffee
> <https://github.com/buildbot/buildbot/pull/1665#discussion_r30468512>:
>
> > @@ -56,6 +56,9 @@ config =
> >              'underscore.string':
> >                  version: "~2.3.3"
> >                  files: 'lib/underscore.string.js'
> > +            'sortable.js':
>
> Could you explain why did you choose this lib over the other available?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1665/files#r30468512>.
>